include/api.php

```diff
@@ -664,18 +664,20 @@ function api_item_get_user(&$a, $item) {
 
 		// Comments in threads may appear as wall-to-wall postings.
 		// So only take the owner at the top posting.
-		if ($item["id"] == $item["parent"])
-			$status_user = api_get_user($a,$item["owner-link"]);
-		else
-			$status_user = api_get_user($a,$item["author-link"]);
-
+		#if ($item["id"] == $item["parent"])
+		#	$status_user = api_get_user($a,$item["owner-link"]);
+		#else
+		
+		$status_user = api_get_user($a,$item["author-link"]);
 		$status_user["protected"] = (($item["allow_cid"] != "") OR
 						($item["allow_gid"] != "") OR
 						($item["deny_cid"] != "") OR
 						($item["deny_gid"] != "") OR
 						$item["private"]);
 
-		return ($status_user);
+		$owner_user = api_get_user($a,$item["owner-link"]);						
+						
+		return (array($status_user, $owner_user));
```

Comment:
I think the docs should be extended now, when the function returns more data then before.